docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+﻿# Projection management
+
+The various gRPC client APIs includes dedicated clients that allow you to manage projections.
+
+:::note
+Currently not all clients fully expose all operations.
+:::
+
+For a detailed explanation of projections, see the [server](/server/v21.2/docs/projections/) documentation.
+
+The full sample code shown here can be found on the respective [clients repositories](https://github.com/eventStore/?q=client)
+
+## Required packages
+<xode-group>
+<xode-block title="C#">
+
+```bash
+dotnet add package EventStore.Client.Grpc.ProjectionManagement --version 21.*
+```
+</xode-block>
+<xode-block title="Java">
+
+```
+# Maven
+<dependency>
+  <groupId>com.eventstore</groupId>
+  <artifactId>db-client-java</artifactId>
+  <version>1.0.0</version>
+</dependency>
+
+# Gradle
+implementation 'com.eventstore:db-client-java:1.0.0'
+
+# SBT
+libraryDependencies += "com.eventstore" % "db-client-java" % "1.0.0"
+```
+</xode-block>
+<xode-block title="NodeJS">
```

Comment:
**FIX:** We should split it into JavaScript and TypeScript code blocks as in the other docs.

Reply:
@ylorph agreed